lib/polyglot.js

```diff
@@ -268,14 +268,18 @@
   // Does the dirty work. Creates a `RegExp` object for each
   // interpolation placeholder.
   function interpolate(phrase, options) {
-    for (var arg in options) {
+	Object.keys(options).forEach(function(arg) {
+    //for (var arg in options) {
       if (arg !== '_' && options.hasOwnProperty(arg)) {
+		// reaplcement value should be escaped to avoid special regex replacement symbols
+		// e.g.: $& or $1
+		var replacement = options[arg].replace ? options[arg].replace(/\$/g, '$$$$') : options[arg];
```

Comment:
Very yes, this in fact required to handle nulls: check for null.replace crashes.  Update to follow.